releasenotes.txt

```diff
@@ -3062,7 +3283,7 @@ Alpha Build 0.3.0.0
 - BatcherImpl and PreparerImpl were combined and code cleaned up thanks to problems found when using Ngpsql (Martijn Boland).
 - ITransaction is now responsible for joining IDbCommand to IDbTransaction instead of IBatcher - if applicable.
 - Modified code to help improve performance of Drivers that don't support multiple Open DataReaders on a single IDbConnection.
-- Fixed bug with hbm2net and VelocityRenderer throwing Exception (Carlos Guzm�n �lvarez & Peter Smulovics).
+- Fixed bug with hbm2net and VelocityRenderer throwing Exception (Carlos Guzm�n �lvarez & Peter Smulovics).
```

Comment:
Som unicode screw-ups

Reply:
VS Code gotcha: Microsoft/vscode#23997, they read a limited number of bytes before guessing the encoding. (At first only 512, they have increased that, but of course if they do not read the whole file they will still screw some cases...) Going to fix by switching the file to UTF8+BOM.